src/libm/sleefdp.c

```diff
@@ -30,6 +30,36 @@ extern const double rempitabdp[];
 #pragma fp_contract (off)
 #endif
 
+#define POLY2(x, c1, c0) mla(x, c1, c0)
```

Comment:
These macros seem to be the same in all `sleef*.c` files. I think it would make sense to merge them in a shared header file (`misc.h`?)